src/helper.ts

```diff
@@ -1,4 +1,4 @@
-import merge from 'deepmerge';
+import * as merge from "deepmerge";
```

Comment:
What is the reason we are importing this way?

Reply:
Since they are not exported as default or member function, ts cannot directly access them, so have to import it in above mentioned manner